stdlib/socket/src/main/ballerina/socket/client_socket.bal

```diff
@@ -85,9 +85,11 @@ public type Client client object {
 #
 # + host - Target service URL
 # + port - Port number of the remote service
+# + readTimeout - Socket read timeout value to be used in milliseconds. Default is 300000 milliseconds (5 minutes)
 # + callbackService - The callback service for the client. Resources in this service gets called on receipt of messages from the server.
 public type ClientConfig record {|
     string host;
     int port;
+    int readTimeout?;
```

Comment:
```suggestion
    int readTimeout = 300000;
```

Reply:
Since readTimeout has a default value, shall we can make it a non-nilable field?